### PR TITLE
Convert Screens to Sass

### PR DESCRIPTION
**Pull request by mistermendez**

### CODE REVIEW

**Comment by mistermendez** [comment]
merge https://github.com/ooyala/html5-skin/pull/250 into this PR before merging this into master

**Comment by mistermendez** [comment]
conflicts merged

File: js/mixins/truncateTextMixin.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+var TruncateTextMixin = {
+  truncateText: function(node, text) {
+    // CSS doesn't support "truncate N lines" so we need to do DOM width
+    // calculations to figure out where to truncate the description
+    var descriptionNode = React.findDOMNode(node);
+    descriptionNode.innerHTML = Utils.truncateTextToWidth(descriptionNode, text);
+  }
+};
+module.exports = TruncateTextMixin;
```
Comment (chibzu) [inline js/mixins/truncateTextMixin.js]: What's the benefit here? Not sure I see the benefit in saving one line of code for another layer of abstraction/misdirection.

File: js/mixins/resizeMixin.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  componentWillUnmount: function() {
+    window.removeEventListener('resize', this.handleResize);
+    window.removeEventListener('webkitfullscreenchange', this.handleResize);
+    window.removeEventListener('mozfullscreenchange', this.handleResize);
+    window.removeEventListener('fullscreenchange', this.handleResize);
+    window.removeEventListener('msfullscreenchange', this.handleResize);
+  }
+};
+module.exports = ResizeMixin;
```
Comment (chibzu) [inline js/mixins/resizeMixin.js]: We should document the implicit contract this mixin creates, which is that there's a handleResize function on the component it's being mixed into.

File: js/mixins/truncateTextMixin.js (hunk 1 above)
Comment (mistermendez) [inline js/mixins/truncateTextMixin.js]: Hi @chibzu, mainly to make it consistent across components.

Some components were using deprecated getDOMNode() or truncating text after the component mounted with state variables, it was easier to consolidate into one function.

And we may need to use this in additional components like discovery

File: js/mixins/resizeMixin.js (hunk 2 above)
Comment (mistermendez) [inline js/mixins/resizeMixin.js]: @chibzu added js docs to mixins

File: js/mixins/truncateTextMixin.js (hunk 1 above)
Comment (chibzu) [inline js/mixins/truncateTextMixin.js]: Ah ha, hadn't seen that getDOMNode was deprecated or that those APIs were volatile. Early adopter woes, I suppose.

**Comment by chibzu** [comment]
Looks good.

**Pull request merged**